### DISCUSSION TITLE
Conflict in commands

### DISCUSSION THREAD

**Discussion by MohammadRezaKashi** [HandyOrg/HandyControl#871] [General]
I have implemented my WPF project by MVVM design pattern so I needed to implement observable class by myself.
Now it has conflicts with handy controls commands.
How can I solve this problem?

**Comment by ghost1372** [comment]
As mentioned in the documentation, all commands are in the  `HandyControl.Tools.Command` namespace
To prevent interference, you can do the following

1. If you do not need this namespace, delete it
2. Use the appropriate namespace when creating your class

```
namespace myproject.mycommand
     public class command {}
```
3. If the above does not solve your problem, you can use the following method
`using DelegateCommand = HandyControl.Tools.Command;`

Replace your class name and namespace
